src/lib/grammar.js

```diff
@@ -80,6 +80,7 @@ const GRAMMAR = {
     'IMPORTANT'     : '!',
     // https://regex101.com/r/mM2vT9/8
     'NAMED'         : '([\\w$]+(?:(?:-(?!\\-))?\\w*)*)',
+    'CSS_VARIABLE'  : '(--(?:(?:-(?!\\-))?\\w*)*)',
```

Comment:
Really not sure about this regex here, so please correct this.

Reply:
It seems to match anything after `--`, even spaces, so I think something is not quite right.  Also can remove the outermost capture group since we don't need to capture any portion of the match I believe

Reply:
https://regex101.com/r/9K9JUE/1

Reply:
I've updated the regex to just `'(--[\\w-]+)'` in https://github.com/acss-io/atomizer/pull/395/commits/5dc7cbc932bc7aebbcecef421bf95ea453a911eb